### DISCUSSION TITLE
2024-02-09-tools

### DISCUSSION THREAD

**Discussion by giscus** [tw93/tw93.github.io#168] [General]
# 我的年度工具折腾之旅 - Tw93

Hi，我是 Tw93，是一个前端工程师，会一点 Swift / Rust / Python，喜欢折腾各种新技术。Share最近趁着过年有空，将一年自己使用的比较不错的 Mac 软件、网站、生活好物给整理了一下，保持着简单易用美观的原则，期待当你有同样选择纠结的时候的一些帮助。

https://tw93.fun/2024-02-09/tools.html

**Comment by MikeYY666** [comment]
哈哈哈哈哈哈 TW，求推荐些RSS 信息源～

**Reply by tw93** [reply]
改天来分享一波

**Reply by Phinsin666** [reply]
推荐些RSS信息源+1 ！

**Comment by dockerytong** [comment]
元素周期表强烈推荐 [ptable](ptable.com)，不仅美观，信息更全面

**Reply by tw93** [reply]
谢谢推荐

**Comment by Y80** [comment]
大佬好，我也在尝试写一些博客文章，但是我很纠结图床。
所以额外关注了下你的图床，你用的是公司的对象存储和 CDN 吗，为什么不考虑使用自己的呢？

**Comment by caijinyc** [comment]
感谢分享！

**Comment by JabinHao** [comment]
大佬太强了

**Comment by aiguo688** [comment]
感谢分享！

**Comment by s-leeq** [comment]
请问大佬有试过 macOS 自带的 `Digital Color Meter` 吗？感觉 `Drop` 和它在功能上没有什么很大的不同诶，貌似就是 `Drop` 多了个储存颜色值的功能？

**Reply by tw93** [reply]
应该是我电脑哪儿设置有问题，其他的取色工具我感觉都不是很准一样... 只有drop的颜色是正确的，所以就...

**Reply by s-leeq** [reply]
我刚刚买了一个看了一下，Drop的颜色值就和Digital Color Meter的sRGB模式数值一样，大佬可以试试看👀

**Comment by zzsqwq** [comment]
感谢分享！这个 Top Calendar 貌似没法在国内的 MAS 中下载了，难受了

**Reply by s-leeq** [reply]
我在马来西亚区也下不了，我还以为是单纯没上架马来区呢😂还专门发邮件去问开发者，但是开发者也没有回应

**Reply by tw93** [reply]
对的，有朋友反馈了，只能等开发者回应了

**Reply by Beeta** [reply]
美国去也下不了，估计是出问题了

**Reply by s-leeq** [reply]
哈啰，更新一下，现在应该可以下载了，开发者给我回邮件说已经重新上架了，可以去看一下～

**Reply by tw93** [reply]
https://apps.apple.com/cn/app/top-calendar/id1544980542?mt=12
可以下载了，看起来应该是之前作者没有注意到

**Comment by Takdrift** [comment]
请问Raycast比Alfred主要强在哪些方面？尤其是大佬工作、生活相关的使用场景，感谢回复！

**Reply by tw93** [reply]
功能更加丰富，比如说你用了Raycast，很多工具软件，比如说窗口的管理、快捷键管理基本上可以不需要了，然后和Alfred我常用的历史剪切板，文件搜索，app 搜索这种该有的都有，免费版本完全够用～

**Reply by Takdrift** [reply]
感谢回复，已经从Alfred换成Raycast了，至少比如想快速打开某个特定网页（比如NAS或者路由器）可以在Raycast上免费实现，还有文件预览的功能。
不过相比Win平台上Listary保存文件时快速定位到已打开文件夹的功能，貌似还没一个完美的Mac平替，不知道大佬在保存文件时有没有快速定位文件夹的好办法。

**Comment by DoubleLazyZ** [comment]
好文推！大佬對於mac記憶體選擇了32GB，會有遇到不夠用的問題嗎，想換設備，但一直在煩惱記憶體的問題

**Reply by tw93** [reply]
我认为完全够用

**Comment by HaoyuRuan** [comment]
我记得楼主是阿里同事吧？

**Reply by tw93** [reply]
yep
